### DISCUSSION TITLE
Region patch is not working

### DISCUSSION THREAD

**Discussion by jaydev-dave** [nextfoam/baram#129] [Q&A]
I am using Fluent with a tetra mesh. When I use a single body (and thus a single region), the simulation works perfectly. However, when I use multiple bodies and multiple regions, it gives an error during patching.

Geometry:  

![Unbenannt](https://github.com/user-attachments/assets/30d887d7-71a6-418b-962b-6ab753f87848)

I've inserted the error bellow:

> /*---------------------------------------------------------------------------*\
> | =========                 |                                                 |
> | \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox           |
> |  \\    /   O peration     | Version:  2312                                  |
> |   \\  /    A nd           | Website:  www.openfoam.com                      |
> |    \\/     M anipulation  |                                                 |
> \*---------------------------------------------------------------------------*/
> Build  : c897e9d1-20240920 OPENFOAM=2312 version=v2312
> Arch   : "LSB;label=32;scalar=64"
> Exec   : C:\Program Files\BARAM\solvers\openfoam\bin\chtMultiRegionSimpleNFoam
> Date   : Oct 16 2024
> Time   : 10:11:46
> Host   : 7SDWXY3
> PID    : 32320
> I/O    : uncollated
> Case   : ---
> nProcs : 1
> trapFpe: Floating point exception trapping enabled (FOAM_SIGFPE).
> fileModificationChecking : Monitoring run-time modified files using timeStampMaster (fileModificationSkew 0, maxFileModificationPolls 20)
> allowSystemOperations : Allowing user-supplied system call operations
> 
> // * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
> Create time
> 
> Create fluid mesh for region fluid for time = 0
> 
> 
> --> FOAM FATAL ERROR: (openfoam-2312)
> Illegal neighbourPatch name inlet_pipe_to_fluid
> Valid patch names are 3(wall_total_w_prism_layers fluid_to_inlet_pipe fluid_to_outlet_pipe)
> 
>     From virtual Foam::label Foam::cyclicAMIPolyPatch::neighbPatchID() const
>     in file AMIInterpolation/patches/cyclicAMI/cyclicAMIPolyPatch/cyclicAMIPolyPatch.C at line 913.
> 
> FOAM exiting
> 

Can anyone suggest how to solve it?

**Comment by kdg5424** [comment]
Hello @jaydevdave97. This is Donggyu from NEXTFOAM.
We developed utility "fluentToFoam" which converts to multi-region in BARAM and OpenFOAM.
But I'm not sure whether it is included in current BARAM or former one (or will be release in next version)
I'll check it to the developers and let you know.

And #130 looks like the same issue. So could you close it?
Thanks for in advance.

Best regards

DonggyuKim

**Reply by jaydev-dave** [reply]
Hey, thanks. Did you check out? Because I'm still have same problem.
